docs/source/cross_compile/config_dev_system.rst

```diff
@@ -133,6 +133,12 @@ the right location.
    supports. For example, the 2018-2019 toolchain is typically referred
    to as the 18.0 version.
 2. Use 7-Zip to extract the contents of the toolchain.
+
+**Note:** Extracting the toolchain requires the ability to create
+symbolic links or the toolchain will not work properly. On Windows,
+this requires that the current user have permissions to create
+symbolic links or that 7-Zip be run as an administrator. 
```

Comment:
We could also use the [`..note::`](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes) decorator here, I suppose.

Reply:
That looks much nicer. I think I'm going to go through and fix up all of our notes to do so in a separate PR.